File: src/main/java/resources/json/JSONExceptionFileNotFound.java
package resources.json;

import org.json.JSONException;

/**
 * JSONExceptionFileNotFound: JSONException hvis en .json-fil ikke kan findes
 * 
 * @author Joakim Bøegh Levorsen, s185023
 */
public class JSONExceptionFileNotFound extends JSONException {
    public static final long serialVersionUID = 1234567893;

    public JSONExceptionFileNotFound() {
        super("File not found");
    }
}

File: src/main/java/resources/json/JSONExceptionFileNotFound.kt
package resources.json

import org.json.JSONException

/**
 * JSONExceptionFileNotFound: JSONException hvis en .json-fil ikke kan findes
 *
 * @author Joakim Bøegh Levorsen, s185023
 */
class JSONExceptionFileNotFound : JSONException("File not found") {
    companion object {
        val serialVersionUID: Long = 1234567893
    }
}
